### PR TITLE
RC1 readme stage updates

### PR DESCRIPTION
**Pull request by dylans**
[skip ci]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,19 +94,19 @@ The following tables contain status information for the packages:
...
+|[dojo/stores](https://github.com/dojo/stores)|[![Build Status](https://travis-ci.org/dojo/stores.svg?branch=master)](https://travis-ci.org/dojo/stores)|[![codecov.io](https://codecov.io/gh/dojo/stores/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/stores/branch/master)|[![npm version](https://badge.fury.io/js/%40dojo%2Fstores.svg)](https://badge.fury.io/js/%40dojo%2Fstores)|RC1|
 |[dojo/streams](https://github.com/dojo/streams)|[![Build Status](https://travis-ci.org/dojo/streams.svg?branch=master)](https://travis-ci.org/dojo/streams)|[![codecov.io](https://codecov.io/gh/dojo/streams/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/streams/branch/master)|[![npm version](https://badge.fury.io/js/%40dojo%2Fstreams.svg)](https://badge.fury.io/js/%40dojo%2Fstreams)|Alpha|
 |[dojo/themes](https://github.com/dojo/themes)|[![Build Status](https://travis-ci.org/dojo/themes.svg?branch=master)](https://travis-ci.org/dojo/themes)|[![codecov.io](https://codecov.io/gh/dojo/themes/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/themes/branch/master)|[![npm version](https://badge.fury.io/js/%40dojo%2Fthemes.svg)](https://badge.fury.io/js/%40dojo%2Fthemes)|Alpha|
```
Comment (agubler) [inline README.md]: guess this should go to RC1?

File: README.md [hunk 2]
```diff
@@ -161,22 +161,22 @@ others who are developing Dojo 2 applications:
...
+|[dojo/grunt-dojo2](https://github.com/dojo/grunt-dojo2/)|[![Build Status](https://travis-ci.org/dojo/grunt-dojo2.svg?branch=master)](https://travis-ci.org/dojo/grunt-dojo2)|[![codecov.io](https://codecov.io/gh/dojo/grunt-dojo2/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/grunt-dojo2/branch/master)|[![npm version](https://badge.fury.io/js/grunt-dojo2.svg)](https://badge.fury.io/js/grunt-dojo2)|RC1|
+|[dojo/grunt-dojo2-extras](https://github.com/dojo/grunt-dojo2-extras/)|[![Build Status](https://travis-ci.org/dojo/grunt-dojo2-extras.svg?branch=master)](https://travis-ci.org/dojo/grunt-dojo2-extras)|[![codecov.io](https://codecov.io/gh/dojo/grunt-dojo2-extras/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/grunt-dojo2-extras/branch/master)|[![npm version](https://badge.fury.io/js/grunt-dojo2-extras.svg)](https://badge.fury.io/js/grunt-dojo2-extras)|RC1|
```
Comment (agubler) [inline README.md]: not even sure what this is used for now days? think it's just dojo.io - do you we should actually even list these on here?

File: README.md [hunk 3]
```diff
@@ -161,22 +161,22 @@ others who are developing Dojo 2 applications:
...
+|[dojo/scripts](https://github.com/dojo/scripts)|[![Build Status](https://travis-ci.org/dojo/scripts.svg?branch=master)](https://travis-ci.org/dojo/scripts)|[![codecov.io](https://codecov.io/gh/dojo/scripts/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/scripts/branch/master)|[![npm version](https://badge.fury.io/js/%40dojo%2Fscripts.svg)](https://badge.fury.io/js/%40dojo%2Fscripts)|RC1|
```
Comment (agubler) [inline README.md]: This is not being used at all yet so shouldn't be RC1

File: README.md [hunk 4]
```diff
@@ -161,22 +161,22 @@ others who are developing Dojo 2 applications:
...
+|Package|CI Status|Code Coverage|npm       |
+|-------|---------|-------------|----------|
+|[dojo/actions](https://github.com/dojo/actions)|[![Build Status](https://travis-ci.org/dojo/actions.svg?branch=master)](https://travis-ci.org/dojo/actions)|[![codecov.io](https://codecov.io/gh/dojo/actions/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/actions/branch/master)| |
+|[dojo/app](https://github.com/dojo/app)|[![Build Status](https://travis-ci.org/dojo/app.svg?branch=master)](https://travis-ci.org/dojo/app)|[![codecov.io](https://codecov.io/gh/dojo/app/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/app/branch/master)| |
+|[dojo/cli-build-webpack](https://github.com/dojo/cli-build-webpack)|[![Build Status](https://travis-ci.org/dojo/cli-build-webpack.svg?branch=master)](https://travis-ci.org/dojo/cli-build-webpack)|[![codecov.io](https://codecov.io/gh/dojo/cli-build-webpack/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/cli-build-webpack/branch/master)|[![npm version](https://badge.fury.io/js/%40dojo%2Fcli-build-webpack.svg)](https://badge.fury.io/js/%40dojo%2Fcli-build-webpack)|
```
Comment (agubler) [inline README.md]: should move this to deprecated section

File: README.md (hunk 2 above)
Comment (dylans) [reply README.md]: not sure...

File: README.md (hunk 4 above)
Comment (dylans) [reply README.md]: This is the deprecated section. :)

Comment (agubler) [reply README.md]: oh yeah lol

**Pull request merged**